docs/csharp/language-reference/operators/remainder-operator.md

```diff
@@ -8,21 +8,25 @@ ms.topic: "article"
 f1_keywords: 
   - "%_CSharpKeyword"
 helpviewer_keywords: 
-  - "modulus operator [C#]"
+  - "remainder operator [C#]"
   - "% operator [C#]"
 ms.assetid: 3b74f4f9-fd9c-45e7-84fa-c8d71a0dfad7
 caps.latest.revision: 15
 author: "BillWagner"
 ms.author: "wiwagn"
 ---
 # % Operator (C# Reference)
-The `%` operator computes the remainder after dividing its first operand by its second. All numeric types have predefined remainder operators.  
+The remainder operator (`%`) computes the remainder after dividing its first operand by its second. All numeric types have predefined remainder operators. 
   
 ## Remarks  
+ The formula `a % b` will always return a value on the range `(-b, b)`, exclusive (it can never return `b` or `-b`).
```

Comment:
*The formula `a % b` will always return* -> *The `a % b` returns*
Should be more clarification here: there are two possible answers in the range `(-b, b)`. Which one is returned?

Reply:
No, there are an infinite number of answers. If `b` is `3` then it can return `-2`, `-1`, `0`, `1`, `2`. Or if non-integer, it can also return values like `2.5`.

Reply:
I meant two possible outputs for fixed a and b. 5 % 3 returns -1 or 2, both values are in the (-3, 3) range.

Reply:
Though, I admit I've misinterpreted this sentence.

Reply:
I think the text added is good. I'd change "return a value" to "return an integer value". That ensures that there are only 2 possible values. Then, I'd add:

>  If `q = a / b` and `r = a % b`, then `a = q * b + r`.

That ensures there is exactly one answer that satisfies the definition.